packages/base/links-to-editor.gts

```diff
@@ -162,8 +163,14 @@ export class LinksToEditor extends GlimmerComponent<Signature> {
         consumingRealm: this.realmURL,
       },
     );
-    if (chosenCard) {
-      this.args.model.value = chosenCard;
+    if (chosenCardResource) {
+      // this resource is our own (tied to this component's lifetime),
+      // but we need to make sure the card is loaded before we can set
+      // the value of the BoxModel
+      await chosenCardResource.loaded;
+      if (chosenCardResource.card) {
+        this.args.model.value = chosenCardResource.card;
+      }
```

Comment:
@ef4 I was able to get rid of references to the `CardResource.loaded` everywhere except in our links-to-editor and links-to-many editor. I tried a bunch of different ideas, but i'm unsure how to eliminate these.